esp-hal/src/soc/esp32/mod.rs

```diff
@@ -1,5 +1,3 @@
-//! # SOC (System-on-Chip) module (ESP32)
```

Comment:
Submit these changes separately please

Reply:
Sorry this was on me, I asked for it - I think given that these aren't actually used anywhere in generated docs, it's okay, but I'm happy to file the PR separately if you think that's better

Reply:
Okay, I'll file a new PR - @playfulFence could you undo the soc documentation changes (sorry, bad advice on my part)